### PR TITLE
First Draft Front End with Testing

### PR DESCRIPTION
**Pull request by ocook123**

### CODE REVIEW

File: src/components/App.jsx [hunk 1]
```diff
@@ -1,49 +1,70 @@
...
+import Employees from './Employees';
+import Welcome from './Welcome';
+import Jobs from './Jobs';
 
 // eslint-disable-next-line no-undef
 
 class App extends React.Component {
   constructor(props) {
     super(props);
+    this.renderPage = this.renderPage.bind(this);
```
Comment (TannerBrunscheon) [inline src/components/App.jsx]: Use Arrow Function

File: src/components/App.jsx [hunk 2]
```diff
@@ -1,49 +1,70 @@
...
     };
   }
 
-  isDesktop = () => {
-    return window.innerWidth > 1024;
-  }
-
-  componentDidUpdate = ()=> {
-    if (this.state.mobileMenuActive) { this.addClass(document.body, 'body-overflow-hidden'); } else { this.removeClass(document.body, 'body-overflow-hidden'); }
+    renderPage() {
```
Comment (TannerBrunscheon) [inline src/components/App.jsx]: renderPage = () => {}

No need to bind this then

File: src/components/CostCodes.jsx [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+        {code: '22-250', desc: 'Roof Drains'},
+        {code: '22-260', desc: 'Sand/Oil Interceptor'}
+      ],
+      activity: false
+    };
+    this.onHideYes = this.onHideYes.bind(this);
+    this.onHide = this.onHide.bind(this);
+    this.delete = this.delete.bind(this);
+    this.descEditor = this.descEditor.bind(this);
+    this.onEditorValueChange = this.onEditorValueChange.bind(this);
```
Comment (TannerBrunscheon) [inline src/components/CostCodes.jsx]: Same as above

**Review by TannerBrunscheon — CHANGES_REQUESTED** [review]
Other thing is please install ESLint plugin on your VSCode and it will highlight. If you do saving on focus change it will auto fix what it can. 

I only did a couple of arrow function changes please change all of them

File: src/components/CostCodes.jsx (hunk 3 above)
Comment (ocook123) [reply src/components/CostCodes.jsx]: Done

**Review by TannerBrunscheon — APPROVED** [review]
Looks really good

**Pull request merged**